src/main/java/org/jabref/logic/exporter/IExportFormat.java

```diff
@@ -38,18 +37,4 @@
     void performExport(BibDatabaseContext databaseContext, String file, Charset encoding, List<BibEntry> entries)
             throws Exception;
 
-    /**
-     * Perform the Export.
-     * Gets the path as a java.nio.path instead of a string.
-     *
-     * @param databaseContext the database to export from.
-     * @param file  the Path to the file to write to.The path should be an java.nio.Path
-     * @param encoding  The encoding to use.
-     * @param entries  A list containing all entries that
-     * should be exported. The list of entries must be non null
-     * @throws Exception
-     */
-    void performExport(BibDatabaseContext databaseContext, Path file, Charset encoding, List<BibEntry> entries)
```

Comment:
Yeah, I wanted to use it, too. However, the method with that signature never gets called which led to an NPE.
In ExportAction only the one with String is called.